google/gax/__init__.py

```diff
@@ -150,8 +163,9 @@ class RetryOptions(
     """Per-call configurable settings for retrying upon transient failure.
 
     Attributes:
-      retry_codes: a list of exceptions upon which a retry should be attempted.
-      backoff_settings: a BackoffSettings object configuring the retry
+      retry_codes (list[string]): a list of exception codes upon which a retry
```

Comment:
the actual codes come from a [proto file](https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto) that defines status codes.  They are not specific to gRPC.